tasks/copy-assets.js

```diff
@@ -28,6 +28,8 @@ gulp.task('copy-assets', ['clean-dist'], function(){
         .pipe(gulp.dest('dist/img/'));
     var files = gulp.src('content/files/*')
         .pipe(gulp.dest('dist/files/'));
+    var files = gulp.src('content/mapping/**')
+        .pipe(gulp.dest('dist/mapping/'));
```

Comment:
As the file has not a `.json` extension, do we need to explicitly send the Content-Type or we don't care?

Reply:
For now, I think we don't care.
1) to make the PIM resilient and not rely on the availability of api.akeneo.com, the meta schemas are also stored in the PIM. When validating, it read from its local version, not the remote one.
2) If a third party developer want to check its schema against the meta schema, opening the url will download the file.